### DISCUSSION TITLE
Confused with Mask and PSO optimization

### DISCUSSION THREAD

**Discussion by CHOC4HOL1C** [AstroBridge/BDLensing#200] [Q&A]
My system, DESIJ1539+1650_F140W, looks quite complicated, and I'm confused if my masks are alright. The lowest Chi^2 value I got was -6.18. I'm probably doing something very wrong. Please help! 
![image](https://github.com/ajshajib/BDLensing/assets/95487275/6b384ca5-f86e-4f1d-8f4a-27da59903340)
![image](https://github.com/ajshajib/BDLensing/assets/95487275/77063750-d7ac-4b38-a865-17f57a179600)
<img width="1065" alt="image" src="https://github.com/ajshajib/BDLensing/assets/95487275/6353bbc9-25a5-44bc-8363-88e82f67aa47">

**Comment by Rajit13** [comment]
Upload the full code file to your repository or share the collab file. It would be better to check in that way.

**Reply by CHOC4HOL1C** [reply]
@Rajit13 can you check my fork of the BDLensing Repository? I cannot directly upload the .ipynb file here.
GitHub Fork link: https://github.com/CHOC4HOL1C/BDLensing
Chi6 Colab Link: https://colab.research.google.com/drive/17tPWSfOXEcXi5adP4MI9L2BY73pY0uFq?usp=sharing
Chi7 COlab Link: https://colab.research.google.com/drive/1xuH4y1fNDSxX5FGrYBPG5Bj5JwfXphy2?usp=sharing

**Comment by rafee-adnan** [comment]
Your masks are alright. As I am seeing from your notebook, you have put your initial value of your Einstein radius `theta_E` as `1.4`. You can see from the observed image, it is more than double than that. Put your lower limit around to `2.8` for `theta_E` and fix your gamma value to 2. These fixes would recreate the arc of your lensing system, and your chi-squared value will decrease.
 
<img width="389" alt="Screenshot 2023-09-08 at 4 13 34 PM" src="https://github.com/ajshajib/BDLensing/assets/76589056/ae4203dd-010b-45b1-bca8-546f1f0f24f2">
